components/ctas/Cta.vue

```diff
@@ -19,7 +19,9 @@ import { Component, Prop } from 'vue-property-decorator'
 @Component
 export default class extends Vue {
   @Prop(String) to
+  @Prop(Boolean) primary
```

Comment:
Considering this is an intermediate refactor for splitting logic from style, I would not consider this as a blocker yet but I'd recommend reviewing this interface. If this is a tristate, I think it is better if the property is only one (for instance, `type`, and values are `primary` (default), `secondary` and `tertiary`). Or, in case you want to preserve the properties separated, we would need some prioritization rule such as "the highest type win" and add `cta--primary` y `cta--secondary`, only if there are no higher types.